cypress/integration/manual-aggregated-sync.spec.js

```diff
@@ -1,27 +1,137 @@
-import { dataTest } from "../support/utils";
+import ManualAggregateSyncPageObject from "../pageobjects/ManualAggregateSyncPageObject";
 
+/**
+ * Database: d2-docker-eyeseetea-2-30-datasync-sender
+ */
 context("Manual aggregated sync", function() {
+    const page = new ManualAggregateSyncPageObject(cy);
+
+    const anyOrgUnit = "Ghana";
+    const anyInstance = "Y5QsHDoD4I0";
+    const anyDataset = "Malaria annual data";
+
     beforeEach(() => {
-        cy.login("admin");
-        cy.visit("/#/sync/aggregated");
-        cy.get(dataTest("headerbar-title")).contains("MetaData Synchronization");
-    });
-
-    it("has the correct title", function() {
-        cy.get(dataTest("page-header-title")).contains("Aggregated Data Synchronization");
-    });
-
-    /**
-     * Database: d2-docker-eyeseetea-2-30-datasync-sender
-     */
-    it("syncs correctly malaria annual data", function() {
-        /**cy.get('[data-test="search"] > div > [aria-invalid="false"]').type("Malaria annual data");
-        cy.get(dataTest("TableCell-data-table-row-0-column-displayname")).contains(
-            "Malaria annual data"
-        );*/
-        cy.get(dataTest("Checkbox-data-table-row-0")).click();
-        cy.get(
-            '[data-test="objects-table-action-button"] > :nth-child(1) > [focusable="false"]'
-        ).click();
+        page.open();
+    });
+
+    it("should have the correct title", function() {
+        page.title.contains("Aggregated Data Synchronization");
+    });
+
+    it("should syncs correctly malaria annual data", function() {
+        page.search(anyDataset)
+            .selectRow(anyDataset)
+            .openSyncDialog()
+
+            .selectOrgUnit(anyOrgUnit)
+            .next()
+
+            .selectAllPeriods()
+            .next()
+
+            .selectAllAttributesCategoryOptions()
+            .next()
+
+            .selectReceiverInstance(anyInstance)
+            .synchronize()
+
+            .syncResults.contains("Success");
```

Comment:
This assertion on attributes breaks the chain, does it make sense to be able to perform those checks as a chainable method?

Reply:
It's true these methods break the chain. Normally this is not a problem because the validation about data usually is the last statement. But if you need chain more actions then the chain is broken. Ok, I'm giving some more thought about this.

Reply:
Done! I have replaced data properties to chainable asserts methods.
The goal is not to break the chain and leave the assert responsibility to the test